src/array_util.rb

```diff
@@ -0,0 +1,43 @@
+class ArrayUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers&.select(&:even?)&.sum
+  end
+
+  def self.two_largest_numbers(numbers)
+    return nil if numbers.nil? ||  numbers.size < 2
```

Comment:
Are you sure this is correct? If the array has only 1 number returning `[n, nil]` is not correct?

Reply:
I tested ```max(n)```` method from Ruby for ```[1].max(2)``` and returns ```1````.
So, I think is ok to return one element.